### ISSUE TITLE
Simplify not working when not on the same VLAN

### ISSUE THREAD

**Issue by HermesHonshappo** [MelvinGr/CandySimplyFi-tool#15]
Hello

Been trying to use the HA integration as well as the command-line tool.
It worked a long time ago, maybe a yea, back when we had to get the key manually. I remember doing this.

About 8 months ago, I re-did all my LAN infra to separate IoT into a different VLAN, it stopped working. I didn't make the connection at the time.

Since then, I reinstalled HA on a new machine recently, and that HA now has 2 interfaces (one on LAN, one on IoT LAN). So HA is now connected again directly to the IoT LAN, without going through the router.

Out of curiosity, I tried again today to install the HA integration.  To my surprise, it worked, whereas it failed repeatedly in the past.

I noticed that the values returned by the integration were wrong (time left in the cycle, etc.), so I wanted to send the JSON.  I reinstalled and compiled on two different machines, which are on the regular LAN, and the command line tool refuses to work:

```
% ./simplyfi 192.168.107.31 getkey
## Candy Simply-Fi tool by Melvin Groenendaal ## 
error: find_xor_key_list, could not find key
```

So in a nutshell: HA on IoT VLAN, integration works and sicovers key on its own. command-line tool tested on 2 machines not directly connected to IoT VLAN, and command-line fails on both machines.

So I'm wondering if there could be an issue in the code where if the machine running the code and the Candy device aren't on the same LAN, it fails.  Does that sound possible?

**Comment by Robertiano** [comment]
The same issue here
